draft-ietf-tls-esni.md

```diff
@@ -842,6 +857,28 @@ example, if a client-facing server services many backend origin hosts, only one
 of which supports some cipher suite, it may be possible to identify that host
 based on the contents of unencrypted handshake messages.
 
+Beyond these primary security and privacy goals, ECH also aims to hide, to some
+extent, (a) whether or not a specific server supports ECH and (b) whether or
+not a specific connection uses or negotiates ECH. In particular:
```

Comment:
"a specific uses or negotiates ECH" -> "ECH was accepted in a particular connection".